sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj

```diff
@@ -9,7 +9,7 @@
       Provides developers with libraries for the updated compute platform under Azure Resource manager to deploy virtual machine, virtual machine extensions and availability set management capabilities. Launch, restart, scale, capture and manage VMs, VM Extensions and more. Note: This client library is for Virtual Machines under Azure Resource Manager.
       Development of this library has shifted focus to the Azure Unified SDK. The future development will be focused on "Azure.ResourceManager.Compute" (https://www.nuget.org/packages/Azure.ResourceManager.Compute/). Please see the package changelog for more information.
     </Description>
-    <Version>45.0.0.0</Version>
+    <Version>46.0.0.0</Version>
```

Comment:
The latest released version is 44.0.0.0  we want this to be 45.0.0.0, including your changes and the CloudService changes

Reply:
I see, okay will revert this version change. Thanks!

Reply:
reverted the change